src/db.rs

```diff
@@ -15,7 +17,48 @@ pub type Pool = r2d2::Pool<ConnectionManager>;
 pub type Connection = PooledConnection<ConnectionManager>;
 
 fn make_tls_connector(danger_accept_invalid_certs: bool) -> io::Result<MakeTlsConnector> {
-    let connector = TlsConnector::builder()
+    let key = "CA_ROOT_FILE";
+    let ca_file = match env::var_os(key) {
```

Comment:
Could you please move repeating parts to a separate function and use `info!` macros then all provided variables are defined?